www-src/vendor/scripts/backbone.statemachine.js

```diff
@@ -0,0 +1 @@
+../../node_modules/backbone.statemachine/backbone.statemachine.js
```

Comment:
I tried but it's not that simple with this library. 
If I put this lib in /module , Backbone.StateMachine intialize itself as running in NodeJS : https://github.com/sebpiq/backbone.statemachine/blob/master/backbone.statemachine.js#L22-L25 , and requires 'backbone' and 'underscore', which you didn't put in /module (for convenience I presume : ).